source/octod/core.d

```diff
@@ -175,12 +175,14 @@ struct HTTPConnection
 
         Params:
             url = GitHub API method URL (relative)
+            accept = optional argument with custom `Accept` header to
+                specify for this one request
 
         Returns:
             Json body of the response. If response is multi-page, all pages
             are collected and concatenated into one returned json object.
      **/
-    Json get ( string url )
+    Json get ( string url, string accept = "" )
```

Comment:
OK, but then there is a difference between `null` and `""` and you explicitly choose to use `""` to not care about which case is used. For me there is an important semantic difference between empty string and null (in set theory it would be the difference between an empty set and void). Is like a 3-state variable. Maybe in this case it's the same, but in other cases `""` could be a valid value and `null` the absence of a value (which could be turn in a default value that can't be expressed in compile-time for example).

In that sense, for me it makes more sense to use `null` to specify "I have no idea, just use whatever you think is best" (while `""` could mean please do really use `""` as the accept string).